scripts/jenkins/cloud/ansible/roles/heat-generator/templates/heat-template.yaml

```diff
@@ -71,8 +72,22 @@ resources:
       network_id: { get_resource: {{ name }}_network }
 {%   if network.cidr is defined %}
       cidr: "{{ network.cidr }}"
+{%       if ( network.cidr | ipv6 ) %}
+      ip_version: 6
+{%          if network.is_conf or network.is_mgmt %}
+      ipv6_ra_mode: dhcpv6-stateful
+      ipv6_address_mode: dhcpv6-stateful
+{%          endif %}
+{%       else %}
+      ip_version: 4
+{%       endif %}
 {%   elif network.allocation_pools is not defined %}
       cidr: { get_param: default_subnet_cidr }
+      ip_version: {{ ipv6 | ternary(6, 4) }}
+{%       if network.is_conf or network.is_mgmt %}
+      ipv6_ra_mode: dhcpv6-stateful
```

Comment:
...besides, I think this will break the IPv4 case. Or is setting these attributes on a v4 subnet permitted ?

Reply:
good point. Best not to include it in ipv4

Reply:
this ends up being yaml file, so hopefully they are just ignored if defined, but better to be safe than sorry. we should probably remove it for ipv4 or all together as you mention below.